x-pack/legacy/plugins/siem/public/pages/detection_engine/create_rule/components/description_step/index.tsx

```diff
@@ -123,6 +155,47 @@ const getDescriptionItem = (
       ];
     }
     return items;
+  } else if (field === 'threats') {
+    const threats: IMitreEnterpriseAttack[] = get(field, value);
+    if (threats.length > 0) {
+      return [
+        {
+          title: label,
+          description: (
+            <ThreatsEuiFlexGroup direction="column">
+              {threats.map(threat => {
+                const tactic = tacticsOptions.find(t => t.name === threat.tactic.name);
+                return (
+                  <EuiFlexItem>
+                    <EuiText grow={false} size="s">
+                      <h5>
+                        <EuiLink href={threat.tactic.reference}>
```

Comment:
Will want these to open in a new window -- I lost all progress on my rule when clicking these links 😭 
```suggestion
                        <EuiLink href={threat.tactic.reference} target="_blank">
```